x-pack/plugins/security_solution/public/entity_analytics/components/risk_score_enable_section.tsx

```diff
@@ -177,7 +136,59 @@ const RiskScoreUpdateModal = ({
   );
 };
 
-export const RiskScoreEnableSection = () => {
+const RiskEngineHealth: React.FC<{ currentRiskEngineStatus?: RiskEngineStatus | null }> = ({
```

Comment:
We were hitting the max complexity of 20 for a given function so I have done a very small refactor here to break out some components